### DISCUSSION TITLE
Guake PPA

### DISCUSSION THREAD

**Discussion by Davidy22** [Guake/guake#1960] [Announcements]
After getting in contact with the maintainer of an [existing Guake PPA](https://launchpad.net/~linuxuprising/+archive/ubuntu/guake) I'd to endorse said PPA as a way to install Guake. They were already keeping up with Guake releases but going forward, they should be even closer to our releases as they'll be kept in the loop on coming releases, and they will be added to official Guake documentation.

Find the PPA here: https://launchpad.net/~linuxuprising/+archive/ubuntu/guake

**Comment by mlouielu** [comment]
Do you think we need an official PPA on Launchpad?

For what I see on Launchpad, LinuxUprising owner is also a member (admin) of Shutter: https://launchpad.net/~shutter, if logix is willing to help, we can migrate them into the official Guake PPA, so the PPA url endorsed in documentation can be `~guake/`, which is less confused to users.

**Comment by Davidy22** [comment]
Oh, I told the guy that just his current PPA was fine, but he said that if we made him admin on a Guake on they can just start uploading right away. We just have to make one if that's what we want to do instead of this.

**Reply by gsemet** [reply]
I am not fond of putting many other as admin on guake, why would it be needed ? Do we need to set up some webhooks?

**Reply by Davidy22** [reply]
By admin I mean admin under a Guake group on launchpad, which doesn't exist yet because I haven't gotten round to it but could happen if we wanted.
